integration-tests/pipelines/e2e-tests-pipeline.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: e2e-tests-pipeline
+spec:
+  params:
+    - description: 'Snapshot of the application'
+      name: SNAPSHOT
+      default: ''
+      type: string
+    - name: e2e_test_namespace
+      default: "build-templates-e2e"
+      description: "namespace where e2e-tests will be executed"
+  tasks:
+    - name: prepare-e2e-tests
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+      taskSpec:
+        params:
+        - name: SNAPSHOT
+        results:
+          - name: custom-source-build-bundle
+            description: "custom bundle for source build pipeline"
+        steps:
+          - name: e2e-tests
+            image: quay.io/susdas/appstudio-utils:with-make
+            env:
+            - name: SNAPSHOT
+              value: $(params.SNAPSHOT)
+            script: |
+              echo "SNAPSHOT: ${SNAPSHOT}"
+              export SOURCE_IMAGE=$(echo -n ${SNAPSHOT} | jq -r .components[0].containerImage)
+              echo "Built image: ${SOURCE_IMAGE}"
+              
+              git clone --branch setup-source-build https://github.com/tisutisu/e2e-tests.git
+              cd e2e-tests/
+              make setup-source-build | tee cmd_output
+              last_line_output=$(tail -n 1 cmd_output)
+              IFS='='
+              read -ra arr <<< "$last_line_output"
+              custom_source_build_bundle=${arr[1]}
+
+              echo -n ${custom_source_build_bundle} | tee $(results.custom-source-build-bundle.path)
+
+    - name: run-e2e-tests
+      params:
+        - name: e2e_test_namespace
+          value: "$(params.e2e_test_namespace)"
+        - name: app_suffix
+          value: "app-integration"
+        - name: ec_pipelines_repo_url
+          value: "https://github.com/redhat-appstudio/build-definitions.git"
+        - name: ec_pipelines_repo_revision
+          value: "main"
+        - name: custom_source_build_bundle
+          value: "$(tasks.prepare-e2e-tests.results.custom-source-build-bundle)"
+      runAfter:
+        - prepare-e2e-tests
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: "https://github.com/tisutisu/build-tasks-dockerfiles.git"
+          - name: revision
+            value: "add-integration-test"
+          - name: pathInRepo
+            value: integration-tests/tasks/e2e-test.yaml
+      timeout: "2h"
```

Comment:
I think we may need to increase the overall pipeline timeout as well, e.g. https://github.com/konflux-ci/build-definitions/blob/a7c03a3dd4bdf14da49a7bbe07c4602e3eeb30c5/.tekton/pull-request.yaml#L261-L263

Reply:
Good catch. Added it now.